<?php

namespace ContainerERRENzm;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getCreateUserService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private 'App\Api\Command\CreateUser' shared autowired service.
     *
     * @return \App\Api\Command\CreateUser
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).''.\DIRECTORY_SEPARATOR.'vendor'.\DIRECTORY_SEPARATOR.'symfony'.\DIRECTORY_SEPARATOR.'console'.\DIRECTORY_SEPARATOR.'Command'.\DIRECTORY_SEPARATOR.'Command.php';
        include_once \dirname(__DIR__, 4).''.\DIRECTORY_SEPARATOR.'src'.\DIRECTORY_SEPARATOR.'Api'.\DIRECTORY_SEPARATOR.'Command'.\DIRECTORY_SEPARATOR.'CreateUser.php';

        $container->privates['App\\Api\\Command\\CreateUser'] = $instance = new \App\Api\Command\CreateUser();

        $instance->setName('app:create-user');

        return $instance;
    }
}
